### ISSUE TITLE
Key Binding Examples

### ISSUE THREAD

**Issue by BrutalSimplicity** [rosshadden/sublime-xpath#19]
Is there any chance key binding examples can be included with the plugin? It would be super-awesome to bind the XPath query with a shortcut, and then the select-all content with another key binding. 

FYI, the latter can be overcome if you use BracketHighlighter. Thank you for mentioning it on your page as that is what directed me to that amazing plugin. In BracketHighlighter you can add support for it by binding to the "select text including brackets key mapping".

It would be cool if this plugin could include some binding examples (ala BracketHighlighter) since the capabilities previously mentioned are already part of the list of commands. I know the basic sublime key-binding mapping structure, but I'm not sure what arguments need to be passed.

**Comment by keith-hall** [comment]
I will add examples and details to the readme, thanks for the suggestion :)

**Comment by keith-hall** [comment]
Hi @BrutalSimplicity, just to let you know, that I have just made a pull request which will alter the way the `goto_relative` command works slightly, to make it more customizable and consistent with the xpath commands.  You can see the changes in [this commit](https://github.com/keith-hall/sublime-xpath/commit/e59195ea8d8e663abd4de9dd618dea66707d9ff4). Basically, where before you could only go to the open tag of a previous sibling for example, you now have the choice - I have split the `direction` out, with a new `self` value and a `goto_element` argument. I believe you are using the `"goto_relative", "args": { "direction": "entire" }` command, which will need to change to `"goto_relative", "args": { "direction": "self", "goto_element": "entire" }` to achieve the same functionality once this change goes live.

**Comment by BrutalSimplicity** [comment]
Oh ok, thanks for the info. I'll remember that once the change goes live. I just wanted to let you know that today your plugin reached a new height of usability, when I opened an Xsd file that was about 500,000 lines. Using the XPath querying enhancements you added, I was able to slice and dice that Xsd at will, pulling out and aggregating codes and descriptions that would have taken hours, if not days, to complete without this tool. Instead, I was able to extract the information in about a half-hour, an incredible feat! So again, thank you for the hard work. The functionality is absolutely remarkable.
